test/users.e2e-spec.ts

```
@@ -20,15 +20,24 @@ describe('UsersController (e2e)', () => {
     await app.close();
   });
 
-  it('/users (POST) - should create a user', () => {
+  it('/users (POST) - should create a user with extra properties', () => {
+    // This test actively exploits the mass assignment vulnerability
     return request(app.getHttpServer())
       .post('/users')
-      .send({ name: 'John Doe', email: 'john.doe@example.com' })
+      .send({ name: 'John Doe', email: 'john.doe@example.com', isAdmin: true })
       .expect(201)
       .expect(res => {
```

ESLint

[error] Replace `res` with `(res)`

(prettier/prettier)